src/sqlparser/tests/sqlparser_common.rs

```diff
@@ -136,53 +136,6 @@ fn parse_update() {
     );
 }
 
-#[test]
-fn parse_update_with_table_alias() {
-    let sql = "UPDATE users AS u SET u.username = 'new_user' WHERE u.username = 'old_user'";
```

Comment:
pg support this (and also `DELETE .. AS ..`, which is not supported by parser now), but maybe let's do it later.